### PR TITLE
Feature/issue 29/get started guide

### LINKED ISSUE #29: Add description into README with Get Started guide
### Description
In context of the project we need to have documentation. JavaDoc in the code and "Get Started" for user.

### Solution
Provide "Get Started" guide in the `README.md` file

### DoD

- [ ] Get Started guide is provided in `README.md` file in the project root

### PR DESCRIPTION
**Pull request by vitshch**
## Checklist before requesting a review
- [x] There are no MR remarks
- [x] Clean and concise code that is easy to read and understand
- [x] Code covered with tests and they are green
- [x] Code API has informative JavaDoc on it
- [x] Rich logging with detailed messages and correct use of log levels

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+```
+#### 4. Build your application by using maven
+```shell
+mvn clean package
+```
+## Provided annotations
+- `@Autowired` to inject dependency into a bean field
+- `@Component` to mark class as a bean
+## Limitations
+- Circular dependencies detection does not supported
```
Comment (mvoronov) [inline README.md]: `Circular dependencies detection is not supported`

File: README.md [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+mvn clean install
+```
+#### 3. Add the Bring Framework dependency into your project `pom.xml`
+```xml
+<dependencies>
+    ...
+    <dependency>
+        <groupId>com.bobocode.blyznytsia</groupId>
+        <artifactId>bring-framework</artifactId>
+        <version>1.0-SNAPSHOT</version>
```
Comment (mvoronov) [inline README.md]: Should we provide release version, because SNAPSHOT versions are those versions that are under development

**Review by mvoronov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marynamelnichuk) [inline README.md]: We would not have issues with circular dependencies since we support only fields injection

**Review by marynamelnichuk — APPROVED** [review]

**Review by Serhiiilnytskyi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vitshch) [reply README.md]: Good point!

**Review by mvoronov — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,178 @@
...
+<p align="center">
+    <img src="doc/bring.png" width="50%" alt="Bring Framework"/>
+</p>
+
+**Bring Framework** is an inversion of control and dependency injection framework. It allows you to declare what objects you need and then it processes configuration, creates required objects, sets dependencies and **brings objects that are ready to use**.
+
+## Main features
+- Configuration
+    - Beans are configured by annotations
+    - Configuration bean definition scannng
```
Comment (financeasist) [inline README.md]: please fix a typo in `scannng`

File: README.md [hunk 4]
```diff
@@ -1,4 +1,178 @@
...
+## Quick start
+There are 2 ways to start with **Bring Framework**
+- Go over basic usage instruction
+- Clone the <a href="https://github.com/bobocode-blyznytsia/bring-framework-demo" target="_blank"/>bring-framework-demo</a> project and check what is provided by Bring Framework
+
+Choose what is more suitable for you.
+
+### Basic usage
+To start new application with **Bring Framework**, you should create the instance of **`BringApplication` class frist**. It accepts the **`packageName`**, which will be scanned for bean components. You can use `public static void main(String[] args)` method for that.
+
```
Comment (financeasist) [inline README.md:70]: typo in **`BringApplication` class frist** - frist -> first

**Review by financeasist — CHANGES_REQUESTED** [review]

**Review by financeasist — APPROVED** [review]

**Pull request merged**